sensor/kubernetes/sensor/options.go

```diff
@@ -19,6 +19,7 @@ type CreateOptions struct {
 	certLoader                         centralclient.CertLoader
 	localSensor                        bool
 	k8sClient                          client.Interface
+	realK8sClient                      client.Interface
```

Comment:
I think the disctinction between these might be very confusing to future readers. "Why is the fake sensor requiring the real client while real sensor does not need one!?" Perhaps instead something like:
```suggestion
	k8sClient                          client.Interface
	introspectionK8sClient             client.Interface
```

Reply:
Yeah that's much better naming